src/member-api.js

```diff
@@ -561,9 +569,23 @@ export class MemberApi extends TypedEmitter {
   }
 
   /**
+   * @overload
+   * @param {object} [opts]
+   * @param {true} [opts.includeLeft]
    * @returns {Promise<Array<MemberInfo>>}
    */
-  async getMany() {
+  /**
+   * @overload
+   * @param {object} [opts]
+   * @param {false} [opts.includeLeft=false]
+   * @returns {Promise<Array<ActiveMemberInfo>>}
+   */
+  /**
+   * List members in the project. By default only active Members and Coordinators are returned
+   * @param {object} [opts]
+   * @param {boolean} [opts.includeLeft=false] Set to true to list removed members
+   */
+  async getMany({ includeLeft = false } = {}) {
```

Comment:
haven't wrapped my head around a potential fix but the type inference isn't working correctly when the param isn't specified:

```tsx
// Raises a type error

/** @type {Array<import('../src/member-api.js').ActiveMemberInfo>} */
const activeMembers = await project.$member.getMany()
```

```tsx
// This works

/** @type {Array<import('../src/member-api.js').ActiveMemberInfo>} */
const activeMembers = await project.$member.getMany({ includeLeft:false })
```

Reply:
I think I just need another overload for where opts isn't provided

Reply:
Fixed!